properties/P000147.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: P000147
+name: P-space
+aliases: 
+  - pseudo-discrete
+refs:
+  - doi: 10.1016/0016-660X(72)90026-8
+    name: A topological view of P-spaces (A. Misra)
+---
+
+The intersection of countably many open sets is open. Defined in {{doi:10.1016/0016-660X(72)90026-8}} with the additional assumption of {P2}.
```

Comment:
Just to be completely clear, how about "... with the additional assumption of {P2}, which is not assumed here".

Reply:
@hpecora1 if you agree, you can add a commit to this effect. There's two ways to do it: use the `github.dev` interface we used earlier today, or you can directly edit this file by going to "view reviewed changes" and using the menu to select "Edit this file".